README.md

```diff
@@ -155,16 +155,16 @@ cd distro/
 make
 ```
 
-After running the `make` command successfully, navigate to `distro/build/dist/bin` folder and run the `./hz` script:  
+After running the `make` command successfully, navigate to `distro/build/dist/hazelcast-4.1-BETA-1/bin` folder and run the `./hz` script:  
```

Comment:
I guess we could use some placeholder to avoid a need to manually update a directory name after the version name changes

Reply:
Do you mean to get the version from somewhere (ex.: pom.xml) in readme, or just use a placeholder such as `distro/build/dist/hazelcast-[HZ_VERSION]/bin`? If you mean the first, I don't know any way to do it. Maybe we can add it to release workflow?

Reply:
If it's just a README, we could simply go for `/dist/{hazelcast.version}/bin` or similar, I think

Reply:
done 👍